packages/pages/src/common/src/project/structure.ts

```diff
@@ -74,7 +73,6 @@ export interface ProjectStructureConfig {
 
 export const defaultProjectStructureConfig: ProjectStructureConfig = {
   filepathsConfig: {
-    templatesRoot: "src/templates",
```

Comment:
Why remove this from the default?

Reply:
I removed this so that I could remove `templatesRoot` from the type definition so people wouldn't be able to set in the plugins config